src/main/java/com/google/cloud/dns/ZoneInfo.java

```diff
@@ -84,6 +89,9 @@
      */
     abstract Builder setNameServers(List<String> nameServers);
 
+    /** Sets the label of this zone. */
+    public abstract Builder setLabels(Map<String, String> labels);
```

Comment:
Adding an abstract method to an abstract is a breaking change and will require us to do a major version bump.

Instead, we can add a default, empty implementation to this class and override in the impl class.